candle-examples/examples/llama_multiprocess/model.rs

```diff
@@ -31,6 +30,16 @@ struct AllReduce {
     comm: Rc<Comm>,
 }
 
+fn flash_attn(
```

Comment:
Probably not much advantage defining this function as we don't support the feature-gate here, so maybe just inline `candle_flash_attn::flash_attn` where this is used.

Reply:
Done.